### PR TITLE
`impl Clone` for {`CacheMe<_>`, `DefaultParseMode<_>`, `Throttle<_>`}

### PR DESCRIPTION
**Pull request by WaffleLapkin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+### Added
+
+- `impl Clone` for {`CacheMe<_>`, `DefaultParseMode<_>`, `Throttle<_>`} ([#75][pr75])
```
Comment (Hirrolot) [inline CHANGELOG.md:12]: Is `<_>` needed? Previously we've been sticking to just `AutoSend`, for instance.

Comment (WaffleLapkin) [reply CHANGELOG.md:12]: I'm not sure. Do you think I should remove it?..

Comment (Hirrolot) [reply CHANGELOG.md:12]: Yes, I think it's just superfluous and makes the changelog harder to maintain.

Comment (WaffleLapkin) [reply CHANGELOG.md:12]: I don't think it's that important, but whatever

Comment (WaffleLapkin) [inline CHANGELOG.md:12]:
```suggestion
- `impl Clone` for {`CacheMe`, `DefaultParseMode`, `Throttle`} ([#75][pr75])
```

**Review by Hirrolot — APPROVED** [review]

**Pull request merged**